src/DayColumn.js

```diff
@@ -97,7 +97,10 @@ class DayColumn extends React.Component {
     const current = getNow()
 
     if (current >= min && current <= max) {
-      const { top } = this.slotMetrics.getRange(current, current)
+      const { top } = this.slotMetrics.getCurrentTimeIndicatorRange(
```

Comment:
since this is only using `top` maybe change this to "getCurrentTimePosition" instead and only return top

Reply:
Makes sense